.docker/source/Dockerfile

```diff
@@ -1,35 +1,37 @@
+
+# syntax = docker/dockerfile:1.3
+
 # ghcr.io/ros-planning/moveit2:${ROS_DISTRO}-source
 # Downloads the moveit source code and install remaining debian dependencies
 
 ARG ROS_DISTRO=foxy
 FROM ghcr.io/ros-planning/moveit2:${ROS_DISTRO}-ci-testing
-
 MAINTAINER Robert Haschke rhaschke@techfak.uni-bielefeld.de
-MAINTAINER Dave Coleman dave@picknik.ai
 
+# Export ROS_UNDERLAY for downstream docker containers
 ENV ROS_UNDERLAY /root/ws_moveit/install
-WORKDIR $ROS_UNDERLAY/../src
+# Environment variable used in instructions on moveit.ros.org website for running clang-tidy
+ENV CATKIN_WS $(realpath $ROS_UNDERLAY/..)
```

Comment:
`CATKIN_WS`?

Reply:
Removed, I think also we should remove it from foxy because I was just picking dockerfile changes.